db/schema.rb

```diff
@@ -11,7 +11,9 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20170908160100) do
+ActiveRecord::Schema.define(version: 20170926101024) do
+
+  add_index "_event_logs_old", ["uid", "created_at"], name: "index_event_logs_on_uid_and_created_at", using: :btree
```

Comment:
I don't think any of these other changes should be in the schema.rb.  I don't know why they're in the production dump, but they're not expected as part of the app, and certainly weren't introduced by this PR.  We might want to write a separate card to explore where these non-migration changes came from and either introduce them to schema.rb, or remove them from the production db.

Reply:
The reason I didn't remove them was because they were necessary to have a complete schema so keeping them and committing them seemed like the most useful thing to do. However, these changes can also be added later as part of a different PR. 

I've added a card here: https://trello.com/c/WzntS0j7/213-signon-fix-schema